### ISSUE TITLE
Update post tags

### ISSUE THREAD

**Issue by janmarthedal** [janmarthedal/janmr.com#49]
- posts/a-kronecker-product-and-vec-operator-identity.md: numerical-analysis, matrix-arithmetic, kronecker-product, vec-operator
- posts/a-simple-sudoku-solver.md: 
- posts/archimedes-twin-circles-illustrated.md: 
- posts/arithmetic-by-geometry.md: arithmetic, geometry, visualizations
- posts/atob-vs-btoa.md:
- posts/bell-numbers.md: 
- posts/bitwise-operators-and-negative-numbers.md: binary-arithmetic, multiple-precision, two-adic-numbers, twos-complement
- posts/book-review-prime-obsession.md: book, review, number-theory, riemann-hypothesis, prime-numbers
- posts/book-review-the-book-of-numbers.md: book, review, popular-mathematics
- posts/book-review-the-pleasures-of-counting.md: book, review, popular-mathematics
- posts/bresenhams-line-algorithm.md: 
- posts/comparing-rational-numbers-without-overflow.md: 
- posts/computing-the-greatest-common-divisor.md: algorithms, cpp, gcd, number-theory, integers, euclids-algorithm, steins-algorithm, euclid
- posts/computing-the-integer-binary-logarithm.md: algorithms, binary-arithmetic, integer-logarithm, integer-binary-logarithm, arithmetic, seminumerical
- posts/continued-fractions-and-continuants.md: algorithms, cpp, continued-fractions, continuants, basic-theory
- posts/cpp-templates-usual-arithmetic-conversions.md: cpp, data-types, programming, generics
- posts/crossed-ladders-problem.md: 
- posts/deriving-closed-form-for-square-pyramidal-numbers.md: 
- posts/entering-the-world-of-web-components.md: 
- posts/equality-of-numbers.md: floating-point-numbers
- posts/evaluation-of-fibonacci-numbers.md: 
- posts/evaluation-of-powers.md: algorithms, exponentiation, cpp, optimality, binary-arithmetic
- posts/finite-difference-discretization-of-1d-laplace.md: numerical-analysis, finite-differences, laplace-operator, matrix-arithmetic
- posts/finite-difference-discretization-of-2d-laplace.md: numerical-analysis, finite-differences, laplace-operator, two-dimensional, matrix-arithmetic, laplace-matrix, sparse-matrix, vec-operator, kronecker-product, kronecker-sum
- posts/fractions-and-circles.md: stern-brocot-tree, rational-numbers, visualizations, geometry
- posts/generating-all-permutations.md: 
- posts/going-functional.md: 
- posts/good-rational-bounds.md: 
- posts/grouping-typesets-with-mathjax.md: 
- posts/implementing-multiple-precision-arithmetic-part-1.md: arithmetic, algorithms, cpp, multiple-precision, programming, seminumerical, data-structures, addition, subtraction, multiplication
- posts/implementing-multiple-precision-arithmetic-part-2.md: arithmetic, algorithms, cpp, multiple-precision, programming, seminumerical, data-structures, division
- posts/infinite-series-involving-sideways-sum.md: 
- posts/leap-year-rules.md: 
- posts/line-line-intersection.md: plane-geometry, geometry, lines
- posts/linear-regression-applied.md: machine-learning, linear-regression, numerical-analysis, curve-fitting, optimization
- posts/linear-regression-basics.md: machine-learning, linear-regression, numerical-analysis, linearity, matrix-arithmetic, optimization, least-squares
- posts/mathjax-2.4-vs-2.5.md: 
- posts/mathjax-katex-and-a-lot-of-math.md: 
- posts/multiple-precision/01-number-representation.md
- posts/multiple-precision/02-addition.md
- posts/multiple-precision/03-subtraction.md
- posts/multiple-precision/04-basic-multiplication.md
- posts/multiple-precision/05-basic-short-division.md
- posts/multiple-precision/06-basic-long-division.md
- posts/neural-networks/01-preface.md: machine-learning, neural-networks
- posts/neural-networks/02-the-model.md: machine-learning, neural-networks, graph-theory
- posts/neural-networks/03-multiple-inputs.md: machine-learning, neural-networks, matrix-arithmetic
- posts/neural-networks/04-the-optimization-problem.md: machine-learning, neural-networks, matrix-arithmetic, error-function, optimization, training-set
- posts/neural-networks/05-gradient-descent.md: machine-learning, neural-networks, function-gradient, gradient-descent, optimization, numerical-analysis
- posts/neural-networks/06-back-propagation-derivation.md: machine-learning, neural-networks, error-function, chain-rule, back-propagation
- posts/neural-networks/07-back-propagation-matrix-style.md: machine-learning, neural-networks, back-propagation, matrix-arithmetic
- posts/neural-networks/08-activation-functions.md: machine-learning, neural-networks, activation-function, sigmoid-activation-function, relu-activation-function
- posts/neural-networks/09-implementation.md: machine-learning, neural-networks, python, numpy, algorithm, matrix-arithmetic
- posts/neural-networks/10-linear-regression.md: machine-learning, neural-networks, linear-regression
- posts/neural-networks/11-digit-recognition.md:machine-learning, neural-networks, ocr, one-hot-encoding, mnist-dataset
- posts/nice-geometric-progression-proof.md: proof, visualizations, infinite-series, equalities, closed-form-expression, geometry, visual-proof
- posts/no-intercept-simple-linear-regression.md: machine-learning, linear-regression, numerical-analysis, curve-fitting, optimization, least-squares, center-of-mass
- posts/nth-bit-of-negative-number.md: 
- posts/on-the-divergence-of-a-geometric-progression-sum.md: proof, infinite-series, divergence, closed-form-expression
- posts/origin-centered-simple-linear-regression.md: machine-learning, linear-regression, numerical-analysis, curve-fitting, optimization, least-squares
- posts/prime-factors-of-factorial-numbers.md: prime-numbers, factorials, prime-factors, equalities
- posts/remembering-trigonometric-addition-formulas.md: equalities, trigonometry, addition, proof, visual-proof, visualizations
- posts/rust-and-iterator-adapters.md: 
- posts/simple-linear-regression.md: machine-learning, linear-regression, numerical-analysis, curve-fitting, optimization, least-squares, simple-linear-regression
- posts/six-circles-theorem-illustrated.md: 
- posts/stackexchange-lists.md: 
- posts/structure-of-a-multiple-precision-library-in-cpp.md: 
- posts/sum-of-all-digit-permutations.md: 
- posts/thales-theorem-illustrated.md: 
- posts/the-1d-laplace-matrix-using-numpy-and-scipy.md: numerical-analysis, finite-differences, laplace-operator, laplace-matrix, numpy, scipy, sparse-matrix, matrix-arithmetic
- posts/the-computer-as-brain-breaker.md: 
- posts/the-game-of-nim.md: game-theory, nim, games, winning-strategy, xor, impartial-game, combinatorial-game-theory, sprague-grundy-theorem, nim-sum
- posts/the-stern-brocot-tree-of-fractions.md: continued-fractions, stern-brocot-tree, binary-tree, rational-numbers
- posts/the-wave-equation-on-the-unit-square.md: numerical-analysis, finite-differences, laplace-operator, scipy, wave-equation, two-dimensional, algorithm, visualizations, matrix-arithmetic, ode-solver
- posts/tiling-with-l-trominos.md: 
- posts/time-budgets-for-the-web.md: 
- posts/twelve-ways-of-counting.md: combinatorics, enumerative-combinatorics, permutations, combinations, set-partitions, integer-compositions, integer-partitions
- posts/typesetting-math-with-html-and-css-fractions.md: 
- posts/useful-properties-of-the-floor-and-ceil-functions.md: floor, ceil, continuous-discrete, inequalities, equalities, logarithm, integers, integer-function, square-root, rational-numbers
- posts/visualizing-the-pythagorean-theorem.md: visualizations, pythagorean-theorem, proof, visual-proof, euclid
- posts/where-did-pi-come-from.md: donald-knuth, catalan-numbers, pi, online-lecture
- posts/wrapping-html-inside-mathml.md:
